src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/finance/PrisonerBalanceResource.kt

```diff
@@ -57,8 +58,9 @@ class PrisonerBalanceResource(
       ),
     ],
   )
-  fun getPrisonerIdentifiers(
+  fun getPrisonerBalanceIdentifiers(
```

Comment:
rename method so unique and doesn't then get a number added to the function e.g. `getPrisonerIdentifiers1RequestConfig`